### PR TITLE
Cluster post config

### PR DESCRIPTION
**Pull request by paigerube14**
Similar to the change in prow the moving of ingress and monitoring pods was not working for 4.18

https://github.com/openshift/release/pull/58111/files#diff-c55d0322ad47a3f1409ecd46a06e619a4b4fad3430bb34da714dd0a25fce293c

Passing job: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/cluster-post-config/125/console
Failed job: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/cluster-post-config/2761/console

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@paigerube14 Is the new code still work for the previous releases?

File: openshift-qe-move-pods-infra-commands.sh [hunk 1]
```diff
@@ -259,14 +254,9 @@ data:
...
-      baseImage: openshift/prometheus-node-exporter
-    kubeRbacProxy:
-      baseImage: quay.io/coreos/kube-rbac-proxy
-    grafana:
-      baseImage: grafana/grafana
-    auth:
-      baseImage: openshift/oauth-proxy
+    metricsServer:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
```
Comment (qiliRedHat) [inline openshift-qe-move-pods-infra-commands.sh:259]: In 4.17 document https://docs.openshift.com/container-platform/4.17/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-monitoring_creating-infrastructure-machinesets
metricsServer also has the tolerations stanza, without the tolerations I think the . However, https://github.com/openshift-qe/ocp-qe-perfscale-ci/pull/658/ did not add it too.
```
    metricsServer:
      nodeSelector:
        node-role.kubernetes.io/infra: ""
      tolerations:
      - key: node-role.kubernetes.io/infra
        value: reserved
        effect: NoSchedule
      - key: node-role.kubernetes.io/infra
        value: reserved
        effect: NoExecute
```

**Comment by paigerube14** [comment]
@qiliRedHat I have no idea why it was added originally. Might have found in some early early documentation

Based on [this run](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-payload-control-plane-6nodes/1851524456908853248) it is properly working for 4.17

**Pull request merged**